x-pack/plugins/actions/server/lib/connector_token_client.test.ts

```diff
@@ -301,30 +302,18 @@ describe('update()', () => {
       },
       references: [],
     });
-    unsecuredSavedObjectsClient.checkConflicts.mockResolvedValueOnce({
-      errors: [
-        {
-          id: '1',
-          error: {
-            error: 'error',
-            statusCode: 503,
-            message: 'There is a conflict.',
-          },
-          type: 'conflict',
-        },
-      ],
-    });
-
-    const result = await connectorTokenClient.update({
-      id: '1',
-      tokenType: 'access_token',
-      token: 'testtokenvalue',
-      expiresAtMillis: expiresAt,
-    });
-    expect(result).toEqual(null);
-    expect(unsecuredSavedObjectsClient.create).toHaveBeenCalledTimes(0);
+    const retryIfConflictsMock = jest.spyOn(allRetry, 'retryIfConflicts');
```

Comment:
Can we add an assertion that `unsecuredSavedObjectsClient.create` was called 3 times?

Can we also add another test that mocks a conflict on the first call and then a success on the second call?

Reply:
added in `retry_on_conflicts.test.ts`